### ISSUE TITLE
dnGrep is not opening when I click on it.

### ISSUE THREAD

**Issue by KrisSchneider** [ScoopInstaller/Extras#9199]
## Bug Report

**Package Name:** [name of package which has bug(s)]
dngrep

### Current Behaviour
dngrep wil not open, it always keeps displaying this message:
![(dnGREP)D8h5j2_09-08-2022](https://user-images.githubusercontent.com/101912712/189203449-af540127-a710-4a59-a167-f9d4d6cc4546.jpg)

Also, the software creates this log:
![(VSCodium)d8m1M4_09-08-2022](https://user-images.githubusercontent.com/101912712/189203802-48805344-8f5d-4c5d-9b7c-89836bd8f5c9.jpg)

### Expected Behaviour
dngrep should open just fine without any problems.

### Additional context/output

Log of dnGrep:
```xml
<logevent time="2022-09-08 12:53:16.0635" level="Error" logger="dnGREP.WPF.App">
  <message>Failure in application startup</message>
  <exception>System.Xml.XmlException: Root element is missing.
   at System.Xml.XmlTextReaderImpl.Throw(Exception e)
   at System.Xml.XmlTextReaderImpl.ParseDocumentContent()
   at System.Xml.Linq.XDocument.Load(XmlReader reader, LoadOptions options)
   at System.Xml.Linq.XDocument.Load(Stream stream, LoadOptions options)
   at dnGREP.Common.GrepSettings.GetFileVersion(String path)
   at dnGREP.Common.GrepSettings.Load(String path)
   at dnGREP.Common.GrepSettings.get_Instance()
   at dnGREP.WPF.App.Application_Startup(Object sender, StartupEventArgs e)</exception>
</logevent>
<logevent time="2022-09-08 12:55:16.3242" level="Error" logger="dnGREP.WPF.App">
  <message>Failure in application startup</message>
  <exception>System.Xml.XmlException: Root element is missing.
   at System.Xml.XmlTextReaderImpl.Throw(Exception e)
   at System.Xml.XmlTextReaderImpl.ParseDocumentContent()
   at System.Xml.Linq.XDocument.Load(XmlReader reader, LoadOptions options)
   at System.Xml.Linq.XDocument.Load(Stream stream, LoadOptions options)
   at dnGREP.Common.GrepSettings.GetFileVersion(String path)
   at dnGREP.Common.GrepSettings.Load(String path)
   at dnGREP.Common.GrepSettings.get_Instance()
   at dnGREP.WPF.App.Application_Startup(Object sender, StartupEventArgs e)</exception>
</logevent>
<logevent time="2022-09-08 12:56:32.2693" level="Error" logger="dnGREP.WPF.App">
  <message>Failure in application startup</message>
  <exception>System.Xml.XmlException: Root element is missing.
   at System.Xml.XmlTextReaderImpl.Throw(Exception e)
   at System.Xml.XmlTextReaderImpl.ParseDocumentContent()
   at System.Xml.Linq.XDocument.Load(XmlReader reader, LoadOptions options)
   at System.Xml.Linq.XDocument.Load(Stream stream, LoadOptions options)
   at dnGREP.Common.GrepSettings.GetFileVersion(String path)
   at dnGREP.Common.GrepSettings.Load(String path)
   at dnGREP.Common.GrepSettings.get_Instance()
   at dnGREP.WPF.App.Application_Startup(Object sender, StartupEventArgs e)</exception>
</logevent>

```
**[Download Link](https://github.com/ScoopInstaller/Extras/files/9529969/Grep_Error_Log.zip)**

### System details

**Windows version:** [e.g. 7, 8, 10]
10

**OS architecture:** [e.g. 32bit, 64bit]
64bit

**PowerShell version:** [output of `"$($PSVersionTable.PSVersion)"`]
7.2.6

**Additional software:** [(optional) e.g. ConEmu, Git]
git, gsudo, Windows Terminal.

#### Scoop Configuration

```json
{
  "lastUpdate": "2022-09-08T11:59:39.0576700-06:00",
  "aria2-warning-enabled": false,
  "aria2-enabled": true,
  "SCOOP_REPO": "https://github.com/ScoopInstaller/Scoop",
  "SCOOP_BRANCH": "master"
}
```

**Comment by issaclin32** [comment]
The problem is caused by that newer versions of *dnGrep* cannot recognize empty XMLs.
It has been fixed in https://github.com/ScoopInstaller/Extras/commit/4088149e7bbc4b1d327d7018a67692efa8d831d6

You can either:
(a) run `scoop uninstall dngrep -p`
or:
(b) delete `(SCOOP DIRECTORY)\persist\dngrep`

and see if this solves the problem

**Comment by KrisSchneider** [comment]
> Thanks for reporting. The problem is caused by that newer versions of _dnGrep_ cannot recognize empty XMLs. It has been fixed in [4088149](https://github.com/ScoopInstaller/Extras/commit/4088149e7bbc4b1d327d7018a67692efa8d831d6)
> 
> You can either: (a) run `scoop uninstall dngrep -p` or: (b) delete `(SCOOP DIRECTORY)\persist\dngrep`
> 
> and see if this solves the problem.

I tried (a), and it worked for me. Thank you, this helped.
